src/PolyhedralGeometry/Groups.jl

```diff
@@ -164,7 +164,16 @@ end
 @doc Markdown.doc"""
     automorphism_group(P::Polyhedron; type = :combinatorial)
```

Comment:
How about an `:action` keyword argument which could be `:vertex` or `:facet`? If this makes no sense or does not work out in practice, just discard my comment.

Reply:
Let me know whether I understood you right.

Reply:
Yes, this is what I meant.